routers/api/v1/api.go

```diff
@@ -723,6 +723,8 @@ func Routes(ctx gocontext.Context) *web.Route {
 		m.Get("/gitignore/templates/{name}", misc.GetGitignoreTemplateInfo)
 		m.Get("/licenses", misc.ListLicenseTemplates)
 		m.Get("/licenses/{name}", misc.GetLicenseTemplateInfo)
+		m.Get("/label_templates", misc.ListLabelTemplates)
```

Comment:
How about "/label/templates"?

Reply:
Hmm, I see 

```
		m.Get("/gitignore/templates", misc.ListGitignoresTemplates)
		m.Get("/gitignore/templates/{name}", misc.GetGitignoreTemplateInfo)
```

and 

```
		m.Get("/licenses", misc.ListLicenseTemplates)
		m.Get("/licenses/{name}", misc.GetLicenseTemplateInfo)
```

I think it's better to unify them to the same naming style. Since 1.20 hasn't been released, no worry about "breaking"

Reply:
This different naming styles are there to match the GitHub API, which uses the same naming. As GitHub has no API for Label templates, we can use what we want here. So I can change it to `/label/templates` if you want.

Reply:
`/label/templates` seems good enough and matches `/gitignore/templates`

ps: could there be some comments on `/gitignore` and `/licenses` to clarify that these paths come from GitHub?